components/login-form.html

```diff
@@ -0,0 +1,41 @@
+<div class="flex justify-center items-center h-screen bg-gray-200 px-6">
+    <div class="p-6 max-w-sm w-full bg-white shadow-md rounded-md">
+        <div class="flex justify-center items-center">
+            <span class="text-gray-700 text-2xl">Login</span>
+        </div>
+
+        <form class="mt-4">
+            <label class="block">
+                <span class="text-gray-700 text-sm">Email</span>
+                <input type="email" class="form-input mt-1 block w-full rounded-md focus:border-indigo-600"
+                />
+            </label>
+
+            <label class="block mt-3">
```

Comment:
```suggestion
            <label for="password" class="block mt-3">
```
Same for the password. 😀